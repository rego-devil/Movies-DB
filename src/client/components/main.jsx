import React from 'react';
import {FilmItem} from './';
import filmList from '../data/films.json';

export const Main = () => (
  <div className="main">
      <div className="wrapper">
	  		<ul className="filmList">
			{
				filmList.map(item => <li key={item.id} className="filmItem"><FilmItem {...item} /></li>)
			}
			</ul>
      </div>
  </div>
)