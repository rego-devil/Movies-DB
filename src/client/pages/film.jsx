import React from 'react';
import { FilmListContainer, CurrentFilmContainer, FilmSortContainer } from '../containers';

export const FilmPage = (props) => {
//   {throw  new Error('I crashed!')}
  return (
    <React.Fragment>
        <div className="header">
            <div className="wrapper">
                <CurrentFilmContainer {...props} />
            </div>
        </div>
        <div className="main">
            <div className="wrapper">  
                {/* <FilmSortContainer />
                <FilmListContainer /> */}
            </div>
        </div>
    </React.Fragment>
  )
}