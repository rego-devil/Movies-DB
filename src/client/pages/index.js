export {MainPage} from './main';
export {FilmPage} from './film';
export {NotFound} from './not-found';