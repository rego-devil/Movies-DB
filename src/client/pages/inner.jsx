import React from 'react';

export const Inner = (props) => (
    <div>charter</div>
)