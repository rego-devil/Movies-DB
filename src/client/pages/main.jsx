import React from 'react';
import {Header, FilmList} from '../components';
import { FilmListContainer, HeaderContainer } from '../containers';

export const Main = (props) => {
//   {throw  new Error('I crashed!')}
  return (
    <React.Fragment>
        <HeaderContainer />
        <div className="main">
            <div className="wrapper">
                <FilmListContainer />
            </div>
        </div>
    </React.Fragment>
  )
}