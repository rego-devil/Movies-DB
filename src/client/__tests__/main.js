import React from 'react';
import { shallow, mount, render } from 'enzyme';
import { Main } from '../pages/main';

describe('Main  component', () => {
  test('renders without crashing', () => {
    const wrapper = shallow(<Main />);
    expect(wrapper).toMatchSnapshot();
  });
});
