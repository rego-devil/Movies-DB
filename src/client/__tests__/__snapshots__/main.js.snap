// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main  component renders without crashing 1`] = `
<React.Fragment>
  <Connect(Header) />
  <div
    className="main"
  >
    <div
      className="wrapper"
    >
      <Connect(FilmSort) />
      <Connect(FilmList) />
    </div>
  </div>
</React.Fragment>
`;
